export const Test = async (_, args) => {
  try {
    return "Hello!"
  } catch (error) {
    throw error;
  }
}